troubleshooting.md

```diff
@@ -12,8 +12,8 @@ it is likely that the `/etc/containers/registries.conf` file is either not insta
 misconfigured.
 
 #### Symptom
-```
-sudo buildah bud -f Dockerfile 
+```console
+$ sudo buildah bud -f Dockerfile 
```

Comment:
Good question @nalind.  I like keeping the sudo here and the highlighting works now with the '$' added.